src/status_im/subs.cljs

```diff
@@ -1576,7 +1576,7 @@
  :<- [:home-items-show-number]
  (fn [[search-filter filtered-chats communities view-id home-items-show-number]]
    (if (or (= view-id :home)
-           (and config/new-ui-enabled? (= view-id :chat-stack)))
+           (and @config/new-ui-enabled? (= view-id :chat-stack)))
```

Comment:
Not related to PR, but a separate small bug reported [here](https://github.com/status-im/status-mobile/pull/13747#discussion_r935414648)